src/admin-views/tribe-options-help.php

```diff
@@ -4,97 +4,70 @@
+			0 => __( 'While the resources above help solve a majority of the issues we see, there are times you might be looking for extra support. If you need assistance with a legitimate bug with the plugin and would like us to take a look please follow these steps:', 'tribe-common' ),
+
+			10 => array(
+				'type' => 'ol',
+
+				sprintf( __( '%s. All of the common (and not-so-common) answers to questions we see are all here. It’s often the fastest path to finding an answer!', 'tribe-common' ), '<strong><a href="http://m.tri.be/18j9" target="_blank">' . __( 'Check our Knowledgebase', 'tribe-common' ) . '</a></strong>' ),
+				sprintf( __( '%s. Testing for an existing conflict is the best start for in-depth troubleshooting. We will often ask you to follow these steps when opening a new thread, so doing this ahead of time will be super helpful.', 'tribe-common' ), '<strong><a href="http://m.tri.be/18jh" target="_blank">' . __( 'Test for a theme or plugin conflict', 'tribe-common' ) . '</a></strong>' ),
+				sprintf( __( '%s. There are very few issues we haven’t seen and it’s likely another user has already asked your question and gotten an answer from our support staff. While posting to the forums is open only to paid customers, they are open for anyone to search and review.', 'tribe-common' ), '<strong><a href="http://m.tri.be/4w/" target="_blank">' . __( 'Search our support forum', 'tribe-common' ) . '</a></strong>' ),
+			),
+
+			20 => __( 'If you have tried the steps above and are still having trouble, you can post a new thread to our open-source forums on WordPress.org:', 'the-events-calendar' ),
+
+			21 => $help->get_plugin_forum_links(),
+
+			22 => __( 'Our support staff monitors these forums once a week and would be happy to assist you there.', 'the-events-calendar' ),
+
+			30 => sprintf( __( 'Looking for more immediate support? %s on our website with the purchase of any of our premium plugins. Pick up a license and you can post directly there and expect a response within 24-48 hours during weekdays.', 'the-events-calendar' ), '<a href="http://m.tri.be/4w/" target="_blank">' . esc_html__( 'We offer premium support', 'the-events-calendar' ) . '</a>' ),
+
+
+			40 => __( 'Please note that all hands-on support is provided via the forums. You can email or tweet at us… ​but we will probably point you back to the forums 😄.', 'tribe-common' ),
+			50 => '<div style="text-align: right;"><a href="http://m.tri.be/18ji" target="_blank" class="button">' . __( 'Read more about our support policy', 'tribe-common' ) . '</a></div>',
+		),
+	),
+
+	array(
+		'id' => 'system-info',
+		'title' => __( 'System Information', 'tribe-common' ),
+		'priority' => 30,
+		'content' => array(
+			__( 'The details of your plugins and settings is often needed for you or our staff to help troubleshoot an issue. Please copy and paste this information into the System Information field when opening a new thread and it will help us help you faster!', 'tribe-common' ),
```

Comment:
This will be escaped later on on our code. (Which I forgot to add, will do an Update)